include/EVT/dev/ADCMux.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef EVT_ADCMUX_HPP
+#define EVT_ADCMUX_HPP
+
+#include <EVT/io/GPIO.hpp>
+
+namespace EVT::core::DEV {
+
+    class ADCMux {
+
+    private:
+        ///Pin States
+        EVT::core::IO::GPIO::State low = EVT::core::IO::GPIO::State::LOW;
+        EVT::core::IO::GPIO::State high = EVT::core::IO::GPIO::State::HIGH;
+
+        ///Control pins
+        IO::GPIO& ctrlGPIO1;
+        IO::GPIO& ctrlGPIO2;
+
+        ///Input from ADC
+        IO::ADC& adc;
+
+    public:
+        /**
+         * Create an instance of ADCMux
```

Comment:
These comments should have an empty line between the summary and the parameters/return values